### ISSUE TITLE
Move ownership to JuliaGeo

### ISSUE THREAD

**Issue by yeesian** [wkearn/RasterIO.jl#1]
I'm curious: will you prefer to deal with just the raster format in this repository? If so, a more suggestive name (e.g. [Rasterio](https://github.com/mapbox/rasterio), or GDALRaster) might be better -- feel free to go wild with the naming! It'll also help you scope the responsibility to something more manageable?
- [x] Provide `test/runtests.jl`
- [x] Decide on scope of repository (and rename accordingly)
- [ ] Register in METADATA
- [ ] Move ownership to JuliaGeo

**Comment by wkearn** [comment]
I think since what I've currently got in here is a solid (but not perfect) interface to raster I/O using GDAL, it's a good idea to clean it up and rename it to something (RasterIO.jl?) and then we have room for a GDAL.jl package which implements an API for a wider range of GDAL functionality.

**Comment by wkearn** [comment]
I want to bump this discussion again. I haven't been super active in JuliaGeo stuff for a while, so I'm happy to transfer this to JuliaGeo or wherever else you would like it.

**Comment by visr** [comment]
Hi Will, are you aware of the more recent developments in [visr/GDAL.jl](https://github.com/visr/GDAL.jl) and [yeesian/ArchGDAL.jl](https://github.com/yeesian/ArchGDAL.jl/)? They currently provide a quite extensive wrapping of GDAL/OGR.

I guess this package either can be updated, or explicitly unmaintained/archived, with links to alternatives. What is your preference?

**Comment by yeesian** [comment]
This package was incredibly helpful in bootstrapping the design for [GDAL.jl](https://github.com/visr/GDAL.jl) and packages that builds upon it, like [ArchGDAL.jl](https://github.com/yeesian/ArchGDAL.jl) which offers a more comprehensive scope of the functionality available in GDAL.

My suggestion at this point will first be decide if (i) you still want to maintain this package, and (ii) what functionality you wish to provide in it. If you don't wish to maintain this package, you won't have to do anything here.

If you wish to maintain a package for rasters, my recommendation is to re-write it to use either GDAL.jl or ArchGDAL.jl for building the gdal binaries, because Martijn and I intend to continue maintaining binaries and updating for subsequent versions of GDAL, and we'll be happy to pool efforts together.

It's also worth thinking about what user experience you'll like to achieve in this package that isn't otherwise provided by other packages, and use that as your guiding philosophy for deciding your priorities regarding this package.

**Comment by wkearn** [comment]
Ah, yes. I hadn't been following the various updates very closely. I'll add a statement about the deprecation of this package to the README and point them to the more modern packages. 

I just noticed that RasterIO is listed here: https://github.com/JuliaGeo/meta/wiki/Existing-Libraries and I don't want anyone to think they should be using this to handle raster data in Julia

**Comment by yeesian** [comment]
Thanks Will! I've updated the wiki accordingly.

**Comment by visr** [comment]
Yeah indeed wanted to echo @yeesian that this package was incredibly helpful in bootstrapping the other efforts, so thanks for that!
